docx/oxml/numbering.py

```diff
@@ -203,6 +208,29 @@ def _next_numId(self):
                 break
         return num
 
+    def set_li_indent(self, para_el, styles, prev_p, ilvl):
```

Comment:
Same here. Some short parameter description would be nice.

Furthermore, maybe (not sure about that) better name would be `set_li_level` as the level is what is important. Indent sounds more like visual indentation while here we are dealing more with a structural nesting level.